files/benchmark_job.yml

```diff
@@ -124,10 +128,23 @@
                                 compiler="http://10.40.0.13/toolchains/${file}"
                         fi
 
+                        if [ -d "${WORKSPACE}/mr-provisioner-client" ]; then
+                            rm -rf "${WORKSPACE}/mr-provisioner-client"
+                        fi
+                        git clone https://github.com/Linaro/mr-provisioner-client.git "${WORKSPACE}/mr-provisioner-client"
+
+                        mr_provisioner_url="http://10.40.0.11:5000"
```

Comment:
these variables are now obsolete, you can use the actual value directly on the mrp-client call below.